src/mappings/UController.graphql

```diff
@@ -0,0 +1,63 @@
+type UControllerOrganization @entity {
+    avatarAddress: ID!
+    nativeToken: Bytes!
```

Comment:
nativeToken: NativeToken!

Reply:
Why do we need another entity? also ERC20 is another contract.